WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -2338,6 +2340,100 @@ public static void resetLastUpdateDate() {
     private class LoadTagsTask extends AsyncTask<Void, Void, ReaderTagList> {
         private final FilteredRecyclerView.FilterCriteriaAsyncLoaderListener mFilterCriteriaLoaderListener;
 
+        private static final String KEY_FOLLOWING = "following";
+        private static final String KEY_DISCOVER = "discover";
+        private static final String KEY_LIKES = "likes";
+        private static final String KEY_SAVED = "saved";
+
+        private class TagInfo {
```

Comment:
Would it maybe be possible to extract this logic to a separate class and just expose `getOrderedTagsList`?